conf/settings.py.example

```diff
@@ -20,6 +20,37 @@ FILE_LOGGING = False
 SYSLOG_LOGGING = False
 STREAM_LOGGING = True
 
+LOGGING = {
```

Comment:
Remove the unneeded `*LOG*` options above. It should be also mentioned somewhere that those specific options are overridden by `LOGGING`.

Reply:
Yeah. I was thinking if we can move all the old logging logic in resultsdb/__init__.py

Reply:
Better to keep this backwards compatible.

Reply:
Ok. Keep it.

Reply:
What I meant is allow the options to be used if LOGGING is not defined but the behavior should be mentioned somewhere.

But to keep the example config file clean, let's remove the old options.

Reply:
Ok. Updated, please check if it is better